filters/builtin/static.go

```diff
@@ -34,13 +36,21 @@ func (spec *static) CreateFilter(config []interface{}) (filters.Filter, error) {
 	}
 
 	webRoot, ok := config[0].(string)
+
 	if !ok {
 		return nil, fmt.Errorf("invalid parameter type, expected string for web root prefix")
 	}
 
 	root, ok := config[1].(string)
+
 	if !ok {
-		return nil, fmt.Errorf("invalid parameter type, expected string for path to root dir")
+		log.Errorf("Invalid parameter type, expected string for path to root dir")
+		return nil, filters.ErrInvalidFilterParameters
```

Comment:
@aryszka what do you think about logging + error? 
This can help in many cases to find what you did wrong, so I am ok with log + error in this case.